### DISCUSSION TITLE
Ability to change debounced value instantly

### DISCUSSION THREAD

**Discussion by giray123** [mantinedev/mantine#4092] [Feature requests]
Can we add a method to useDebouncedValue for changing the value instantly?

```js
const [debounced, cancel, changeImmediately] = useDebouncedValue(value, 200);
```

Usecase example: get debounced value from an input field to filter a large array, reset the debounced value with a "clear filters" button without the delay.

**Comment by rtivital** [comment]
You are welcome to submit a PR with this feature
